grudge/dt_utils.py

```diff
@@ -158,7 +159,7 @@ def dt_non_geometric_factors(
 
 @memoize_on_first_arg
 def h_max_from_volume(
-        dcoll: DiscretizationCollection, dim=None, dd=None) -> float:
+        dcoll: DiscretizationCollection, dim=None, dd=None) -> Any:
```

Comment:
Actually I feel like type annotating with `Any` is not very helpful. In fact, I feel it just sorta raises more questions. I'd rather just remove the return annotation (at least for `h_max` and `h_min` here).